modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -108,28 +110,37 @@ $ oc get configmap/coreos-bootimages \
 	  -o jsonpath='{.data.stream}' | jq \
 	  -r '.architectures.<arch>.images.aws.regions."<region>".image'
 ----
-<5> Specify an ARM64 supported machine type. For more information, refer to "Tested instance types for AWS 64-bit ARM"
-<6> Specify the zone, for example `us-east-1a`. Ensure that the zone you select offers 64-bit ARM machines.
-<7> Specify the region, for example, `us-east-1`. Ensure that the zone you select offers 64-bit ARM machines.
+<5> Specify a machine type that aligns with the CPU architecture of the chosen AMI. For more information, see "Tested instance types for AWS 64-bit ARM"
+<6> Specify the zone. For example, `us-east-1a`. Ensure that the zone you select has machines with the required architecture.
+<7> Specify the region. For example, `us-east-1`. Ensure that the zone you select has machines with the required architecture.
 --
 
+. Create the compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name> <1>
+----
+<1> Replace `<file_name>` with the name of the YAML file with compute machine set configuration. For example: `aws-arm64-machine-set-0.yaml`, or `aws-amd64-machine-set-0.yaml`
```

Comment:
I would add a period at the end of the second sentence:
```suggestion
<1> Replace `<file_name>` with the name of the YAML file with compute machine set configuration. For example: `aws-arm64-machine-set-0.yaml`, or `aws-amd64-machine-set-0.yaml`.
```

Reply:
Updated as per the suggestion.